00-HTML-CSS-basics/ex 2/2.3/index.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html class="no-js" lang="">
+<head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="x-ua-compatible" content="ie=edge"/>
+    <meta name="author" content="Maximiliano Tulian"/>
+    <meta name="description" content="Exercise 2.2" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Basics CSS</title>
+
+    <link rel="stylesheet" href="css/normalize.css"/>
```

Comment:
**Minor:** When selfclosing a tag try to  leave one white space between the last attribute and the `/>` :P. No need to fix